tests/tests_lvm_multiple_disks_multiple_volumes.yml

```diff
@@ -58,7 +58,7 @@
         storage_pools:
             - name: phi
               disks: "{{ unused_disks }}"
-              volumes: []
               state: absent
+              volumes: []
```

Comment:
what's the purpose of this change?

Reply:
I guess the "state: absent" was belong to storage_pools, so I changed its position before "volumes:[]", it will be more readable, just like bellow test cases:
tests_change_fs_use_partitions.yml
tests_change_fs.yml
tests_change_mount.yml
tests_create_lvm_pool_then_remove.yml
tests_create_partition_volume_then_remove.yml
tests_disk_errors.yml
tests_filesystem_one_disk.yml
tests_lvm_errors.yml
tests_lvm_one_disk_multiple_volumes.yml
tests_lvm_one_disk_one_volume.yml
tests_remove_mount.yml

Thanks
Yi